### DISCUSSION TITLE
Suggested IDE for pyg project developers.

### DISCUSSION THREAD

**Discussion by Gibyeng** [pyg-team/pyg-lib#295] [Q&A]
I am currently using vscode to do some tests (coding) on pyg, but there are many "header_file_not_find" problems when I use the clang extension in vscode for code highlighting, which is quite anonying. I am curious what IDEs or tools the developers are using. Any suggestions?

**Comment by rusty1s** [comment]
@akihironitta Any opinion on this? Can we somehow mitigate the "header_file_not_find" problem?

**Reply by akihironitta** [reply]
Hi @Gibyeng, we could have a shared vscode config just like PyTorch does https://github.com/pytorch/pytorch/tree/main/.vscode. Let me have a look and add it to the repository.

**Reply by Gibyeng** [reply]
That is awesome. Thanks.
